tests/e2e/provision_with_multiple_zones.go

```diff
@@ -158,4 +159,64 @@ var _ = ginkgo.Describe("[csi-topology-vanilla] Topology-Aware-Provisioning-With
 		pv = nil
 
 	})
+
+	/*
+		Provisioning with multiple zones and regions specified in the Storage Class for non-shared datastore
```

Comment:
Provisioning volume using storage policy with multiple zone and region details in the allowed topology

Reply:
done